src/app/find-help/accommodation-search/accommodation-search-form/accommodation-search-form.component.ts

```diff
@@ -1,22 +1,34 @@
-import { Component, EventEmitter, OnInit, Output } from '@angular/core';
-import { Location } from '@app/core/api';
-import { StatementAnchors } from '@app/shared/models';
+import { Component, EventEmitter, OnDestroy, OnInit, Output, ViewChild } from '@angular/core';
+import { NgForm } from '@angular/forms';
 import { ActivatedRoute, Params, Router } from '@angular/router';
+import { Location } from '@app/core/api';
 import { StoreUrlService } from '@app/core/store-url';
-import { LocalStorageKeys } from '@app/shared/models';
+import { LocalStorageKeys, StatementAnchors } from '@app/shared/models';
 import { SortingFieldName, SortingOrder } from '@app/shared/models/sortingOrder.model';
+import { Subscription } from 'rxjs';
 
 export interface AccommodationQuery {
   location?: Location;
   capacity?: number;
 }
 
+const cleanForm = {
```

Comment:
to jest wymagane, czy tak bardziej chciałeś żeby było jasno pokazane, że to clearujemy?

Reply:
IMO wymagane, w przypadku wysłania tylko podstawowych queryParams i strategii 'merge', to te, które występowały wcześniej a nie były kasowane i zostawały w URLu, wtedy po odświeżeniu pomimo wyczyszczenia formularza, zostały wbijane w niego poprzednie wartości.